build.sbt

```diff
@@ -216,9 +216,24 @@ lazy val parsers = crossProject(JSPlatform, JVMPlatform, NativePlatform)
       )
     })
   )
-  .jsSettings(
+  .jsConfigure(
+      _.enablePlugins(NpmPackagePlugin)
+  ).jsSettings(
     commonJsSettings,
-    scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) }
+    scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) },
+    npmPackageVersion := "4.6.1",
```

Comment:
We'll remove this field by default, `version` in sbt will be used :+1: (This is why I'm using `sbt-npm-package`)

https://github.com/davenverse/sbt-npm-package/blob/a638ce6933f39a02ebe721124229e27dba6d63a0/core/src/main/scala/io/chrisdavenport/npmpackage/sbtplugin/NpmPackagePlugin.scala#L204-L209